.github/workflows/alternative-ci-runners-4.yml

```diff
@@ -13,37 +13,41 @@ permissions:
   contents: read
 
 jobs:
-  docs-lint-on-northflank-kata-remote-engine:
+  docs-lint-on-nf-type-01-0-18-7-remote-engine:
```

Comment:
What are your thoughts @fidencio on removing `0-18-7` from the job name? Keeping this means that we will have a new variant in our metrics that will make it harder to visualise trends.

Reply:
That works for me, let me update the PR once I have this set up on my side (for all the other runners as well)